infra/terraform/test-org/org/locals.tf

```diff
@@ -108,11 +108,11 @@ locals {
       topics      = join(",", [local.common_topics.compute, local.common_topics.containers])
     },
     {
-      name         = "terraform-ecommerce-microservices-on-gke"
```

Comment:
We should check why the linter didn't catch originally.

Reply:
It was my bad, I already had applied the changes in this PR locally so I merged the other `ecommerce-microservices-on-gke` PR and applied without checking lint.

Reply:
No worries @bharathkkb  :) It was just a reminder to myself to investigate as we've had some linter report pass when they actually failed, just happy that wasn't the situtation.